.lintstagedrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "*.{ts,tsx,css,json}": ["prettier --check"],
```

Comment:
Currently limiting to prettierify only these files, but script in `package.json` has no filters. Changing this to the same, just a moment.
"*"